crates/precompile/src/bls12_381_const.rs

```diff
@@ -0,0 +1,63 @@
+pub const G1_ADD_ADDRESS: u64 = 0x0b;
+pub const G1_ADD_BASE_GAS_FEE: u64 = 375;
+pub const G1_ADD_INPUT_LENGTH: usize = 256;
+pub const G1_MSM_ADDRESS: u64 = 0x0c;
+pub const G1_MSM_BASE_GAS_FEE: u64 = 1200;
```

Comment:
This is wrong, should be `12000.` Off by 10

Reply:
ah @holiman nice catch , for this , I have raised [pr](https://github.com/bluealloy/revm/pull/2100)